src-cpp/rdkafkacpp.h

```diff
@@ -2147,8 +2283,8 @@ class RD_EXPORT Producer : public virtual Handle {
                              int msgflags,
                              void *payload, size_t len,
                              const void *key, size_t key_len,
-                             int64_t timestamp,
-                             void *msg_opaque) = 0;
+                             int64_t timestamp, void *msg_opaque,
```

Comment:
Must not change existing APIs since that will break the API stability guarantee

Reply:
Noted thank you, I'll overload the method